test/sensitivities/linalg/generic.jl

```diff
@@ -2,17 +2,28 @@
 
     let ϵ_abs = 1e-6, c_rel = 1e6, N = 5, rng = MersenneTwister(123456)
 
+        # Generate random test quantities for specific types.
+        ∇Arrays = Union{Type{∇RealArray}, Type{ArrayOr∇Real}}
+        trandn(rng::AbstractRNG, ::∇Arrays) = randn(rng, N, N)
+        trandn(rng::AbstractRNG, ::Type{∇Real}) = randn(rng)
+        trand(rng::AbstractRNG, ::∇Arrays) = rand(rng, N, N)
+        trand(rng::AbstractRNG, ::Type{∇Real}) = rand(rng)
+
+        # Get the identity associated to an object.
+        id(::Any) = I
+        id(::Number) = I.λ
```

Comment:
This seems redundant. You can just use the `one` function.

Reply:
Ah, I did not know of `one`. Removed with the latest push.